SimpleBrowser/Network/IHttpWebRequest.cs

```diff
@@ -34,5 +34,6 @@ public interface IHttpWebRequest
         string Referer { get; set; }
         Uri Address { get; }
         string Host { get; set; }
+        System.Security.Cryptography.X509Certificates.X509CertificateCollection ClientCertificates { get; set; }

```

Comment:
Add System.Security.Cryptography.X509Certificates to the usings at the top of the file. Remove the namespace from here.